hnn_core/parallel_backends.py

```diff
@@ -688,31 +684,34 @@ def simulate(self, net, tstop, dt, n_trials, postproc=False):
             The integration time step of h.CVode (ms)
         n_trials : int
             Number of trials to simulate.
-        postproc: bool
+        postproc : bool
             If False, no postprocessing applied to the dipole
 
         Returns
         -------
-        dpl: list of Dipole
+        dpl : list of Dipole
             The Dipole results from each simulation trial
         """
 
         # just use the joblib backend for a single core
         if self.n_procs == 1:
+            print("MPIBackend is set to use 1 core: tranferring the "
+                  "simulation to JoblibBackend....")
```

Comment:
Technically it's not even the JoblibBackend: https://github.com/jonescompneurolab/hnn-core/blob/de14d0cd7a0805a4f6829cd4e03bd4a1faac4aa3/hnn_core/parallel_backends.py#L511-L513 but I guess this is fine.

Reply:
Is it a huge pain to change the naming? Or is it even worth it when joblib is synonymous with "embarrassingly parallel" while being much shorter to type

Reply:
It transfers to `JoblibBackend`, however, it doesn't actually use Joblib, if that's what you mean @jasmainak?

@ntolley sorry, I don't follow. Which name are you suggesting we change?

Reply:
yep that's exactly what I mean